src/document/document.ts

```diff
@@ -234,9 +238,16 @@ export class Document implements Observable<DocEvent> {
     logger.debug(`trying to apply ${changes.length} remote changes`);
 
     if (logger.isEnabled(LogLevel.Trivial)) {
-      logger.trivial(changes.map((change) =>
-        `${change.getID().getAnnotatedString()}\t${change.getAnnotatedString()}`
-      ).join('\n'));
+      logger.trivial(
+        changes
+          .map(
+            (change) =>
+              `${change
+                .getID()
+                .getAnnotatedString()}\t${change.getAnnotatedString()}`,
+          )
+          .join('\n'),
+      );
```

Comment:
This seems harder to read. We need to find a way to prevent a newline per chained call.
https://eslint.org/docs/rules/newline-per-chained-call#ignorechainwithdepth

Reply:
https://github.com/prettier/prettier-eslint-cli/issues/109#issuecomment-506630629